src/master-playlist-controller.js

```diff
@@ -806,6 +811,7 @@ export class MasterPlaylistController extends videojs.EventTarget {
   blacklistCurrentPlaylist(error = {}) {
     let currentPlaylist;
     let nextPlaylist;
+    let isFinalRendition = this.masterPlaylistLoader_.isFinalRendition_();
```

Comment:
I think if you move it closer you can also remove the check for `loader.master` in `loader.isFinalRendition_`